addons/l10n_ar/models/account_move.py

```diff
@@ -25,7 +25,7 @@ def _l10n_ar_get_document_number_parts(self, document_number, document_type_code
         ' identify the type of responsibilities that a person or a legal entity could have and that impacts in the'
         ' type of operations and requirements they need.')
 
-    l10n_ar_currency_rate = fields.Float(copy=False, digits=(16, 6), readonly=True, string="Currency Rate")
+    l10n_ar_currency_rate = fields.Float(copy=False, readonly=True, string="Currency Rate")
```

Comment:
**The main issue being resolved:**
When rounding, in Odoo we round up, but in the PDF we truncate the value.
We need to have the same on both sides

We actually treated this as an improvement and decided to target master as discussed with @jco-odoo last week.
But do you think that will be ok to apply this change from previous versions? We had a PR targeting V16.0 https://github.com/odoo/odoo/pull/136327 and we decided to close it and open this one.

Let me know your thoughts @zaoral on this one, or if you a better/nicer issue description - and it could be included in the PR and commit message.

Thanks

Reply:
> We need to have the same on both sides

Well then we can use the same rounding method on both sides.

> We actually treated this as an improvement and decided to target master as discussed with @jco-odoo last week

Is this a bug or an improvement? The commit message suggests a (bug)fix.

> But do you think that will be ok to apply this change from previous versions?

I don't see any reason why not, why did you change?

> We had a PR targeting V16.0 https://github.com/odoo/odoo/pull/136327 and we decided to close it and open this one.

No need to open a new PR to re-target.

> it could be included in the PR and commit message.

Yes please :)

Reply:
@zaoral please help me with the improvement of the PR and Commit message to be more descriptive, and to add your comments on top of mine.

@jco-odoo also if you want to add your comments on why we moved this to master (you can check my last comments on this thread)

Thanks!